R/importWaterML1.r

```diff
@@ -244,7 +244,12 @@ importWaterML1 <- function(obs_url,asDateTime=FALSE, tz=""){
           obsDF <- full_join(obsDF, valParentDF, by = c("dateTime","tz_cd"))
         }
       }else{
-        obsDF <- data.frame()
+        #need column names for joining later
+        obsDF <- data.frame(dateTime=character(0), tz_cd=character(0), stringsAsFactors = FALSE)
+        if(asDateTime){
+          obsDF$dateTime <- as.POSIXct(obsDF$dateTime)
```

Comment:
I think it would maintain the time zone from the data, based on my little experiments.  But wouldn't hurt to add it here also.  

time zone is set in the parse_date_time section (line 212)